### DISCUSSION TITLE
Tips on contributing

### DISCUSSION THREAD

**Discussion by f-lombardo** [symfony/symfony#47334] [Q&A] [answered]
Hi all! 
I think that this PR introduced a small bug in 5.1.10: https://github.com/symfony/symfony/pull/46525
I'd like to try to solve it by myself and than proposing a PR.  Do you think I should start working from https://github.com/symfony/symfony/tree/v5.1.11? Otherwise what is the correct way of starting a PR?

Thanks in advance.

Franco

**Comment by MatTheCat** [comment, marked as answer]
Hi!

The PR you mention has been merged in the 5.4 branch so there is no way it affected 5.1.

Anyway, you must base your branches on another, but `v5.1.11` is a tag so it would not work. Bugs must be fixed on the oldest maintained branch which contains them. That means if you find a bug introduced on `v5.1.10` you have to work on the 5.4 branch (see https://symfony.com/releases for the list of maintained branches).

You can read https://symfony.com/doc/current/contributing/index.html (and more specifically https://symfony.com/doc/current/contributing/code/pull_requests.html#choose-the-right-branch) to get started.

**Reply by f-lombardo** [reply]
I'm sorry, I made a mistake in my previous post, the affected version is 5.4.10 :-(
Thank you very much for your help!
